tests/unit/tooltip/helper.js

```diff
@@ -14,10 +14,8 @@ define( [
 						timers = jQuery.timers;
 
 					jQuery.fx.stop();
-					var x = false;
```

Comment:
This looks like a breaking change. Is variable `x` not in use?

Reply:
It’s not used, that’s why ESLint complained. It’s only written to but never read.

Reply:
Ahh ok. Strange enough nobody noticed before...